temporalcli/commands.schedule_test.go

```diff
@@ -0,0 +1,361 @@
+package temporalcli_test
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"regexp"
+	"time"
+
+	"github.com/temporalio/cli/temporalcli"
+	"go.temporal.io/sdk/workflow"
+)
+
+func (s *SharedServerSuite) schedId() string   { return s.testVar("sched") }
```

Comment:
```suggestion
func (s *SharedServerSuite) schedID() string   { return s.testVar("sched") }
```

But I don't understand this compared to just making vars in your tests as needed (or returning an ID from `createSchedule` or whatever)